### PR TITLE
Fixes #431

### LINKED ISSUE #431: Generate a python library for the project
**I'm submitting a ...**
- [ ] bug report
- [x] feature request

Generate a python library for the project. So that it can be used from the cli also.

I would be like to work on this. Views @vaibhavsingh97 @djmgit @S2606

### PR DESCRIPTION
**Pull request by yashLadha**
Fixes #431 
This is a demo PR. For discussing the functionality for the package. This package will get published to PyPI and can be used by other developers in their code.

### CODE REVIEW

**Comment by yashLadha** [comment]
Views @vaibhavsingh97 @djmgit @S2606 @gabru-md . This PR is not the implementation, it just about discussing the options that to be included in the library.

**Comment by yashLadha** [comment]
optim

**Comment by raju249** [comment]
Nice One @yashLadha 👍

File: app/cli/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+badgeyay.egg-info/
```
Comment (abishekvashok) [inline app/cli/.gitignore]: Can't we include this in the gitignore at the root?

**Comment by yashLadha** [comment]
Sure...I am working on completing it as soon as possible.

**Comment by abishekvashok** [comment]
Sure take your time.

File: app/cli/.gitignore (hunk 1 above)
Comment (yashLadha) [reply app/cli/.gitignore]: Actually this library will be placed  in root folder. As i will complete the code, will surely put the library in the root folder, or if possible in a different branch or repo? May be?

**Comment by abishekvashok** [comment]
Can you add the build instructions of the library to Travis ci as well. So that we can test that components as well.

**Comment by yashLadha** [comment]
Sure @abishekvashok

**Comment by yashLadha** [comment]
anim-optim

**Comment by mariobehling** [comment]
Ok, good job! Please outline in follow-up issues what else needs to be done to separate front-end and backend. Thanks a lot :-)

**Comment by abishekvashok** [comment]
@mariobehling note that #435 makes up the back end.

**Comment by yashLadha** [comment]
Sure @mariobehling. Will make later issues for the CLI. Also from which
account should I publishing this library. Personal or there is some id of
community.

**Pull request merged**